HLTrigger/Configuration/python/HLT_75e33/tasks/RawToDigiTask_cfi.py

```diff
@@ -1,17 +1,17 @@
 import FWCore.ParameterSet.Config as cms
 
-from ..modules.ecalPreshowerDigis_cfi import *
-from ..modules.hcalDigis_cfi import *
+from ..modules.hltEcalPreshowerDigis_cfi import *
+from ..modules.hltHcalDigis_cfi import *
 from ..modules.hgcalDigis_cfi import *
 from ..modules.muonCSCDigis_cfi import *
 from ..modules.muonDTDigis_cfi import *
 from ..modules.muonGEMDigis_cfi import *
 from ..tasks.ecalDigisTask_cfi import *
```

Comment:
```suggestion
from ..tasks.hltEcalDigisTask_cfi import *
```
Now this is needed, I suppose.